src/main/java/edu/cornell/kfs/tax/dataaccess/impl/TaxProcessingDaoJdbc.java

```diff
@@ -242,28 +248,31 @@ public List<EnumMap<TaxStatType,Integer>> doInConnection(Connection con) throws
                         // Get the results.
                         rs = selectStatement.executeQuery();
                         // Let the builder iterate over the results and insert new transaction detail rows as needed.
-                        builder.buildTransactionRows(rs, insertStatement, summary);
+                        builder.buildRawTransactionRows(rs, rawTransactionInsertStatement, summary);
                         
-                        // Close the result set and SELECT prepared statement to prepare for the second pass.
+                        // Close the result set, SELECT prepared statement and INSERT into first pass (raw) table to prepare for the second pass.
                         rs.close();
                         selectStatement.close();
                         
-                        // Setup retrieval statement for second pass.
-                        selectStatement = con.prepareStatement(builder.getSqlForSelectingCreatedRows(summary),
+                        // Setup retrieval statement for second pass. SELECT needs to obtain data from first pass (raw) transaction details table
+                        selectRawTransactionStatement = con.prepareStatement(builder.getSqlForSelectingCreatedRows(summary),
                                 ResultSet.TYPE_FORWARD_ONLY, ResultSet.CONCUR_UPDATABLE);

```

Comment:
Since you modified the builder classes to no longer use the `ResultSet` itself for updating or deleting rows, you may be able to remove the updatable-result-set configuration here.